blog/models.py

```diff
@@ -1,3 +1,133 @@
+from django import forms
 from django.db import models
 
-# Create your models here.
+from modelcluster.fields import ParentalKey, ParentalManyToManyField
+from wagtail.models import Page, Orderable
+from wagtail.fields import RichTextField, StreamField
+from wagtail.admin.panels import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.search import index
+from wagtail import blocks
+from modelcluster.contrib.taggit import ClusterTaggableManager
+from taggit.models import TaggedItemBase
+from wagtail.snippets.models import register_snippet
+from wagtail.images.blocks import ImageChooserBlock
+
+from wagtail.api import APIField
+from rest_framework import serializers
+
+@register_snippet
+class Author(models.Model):
+    name = models.CharField(max_length=255)
+
+    panels = [
+        FieldPanel('name'),
+    ]
+
+    api_fields = [
+        APIField('name'),
+    ]
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name_plural = 'Authors'
+
+@register_snippet
+class Category(models.Model):
+    name = models.CharField(max_length=255)
+
+    panels = [
+        FieldPanel('name'),
+    ]
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name_plural = 'Categories'
```

Comment:
```suggestion
        verbose_name_plural = 'categories'
```

The capital is something Django can generate on its own.